osidb/sync_manager.py

```diff
@@ -53,84 +53,92 @@ def update_synced_links(self):
         """
         raise NotImplementedError("Update synced links not implemented.")
 
-    def schedule(self):
+    @classmethod
+    def schedule(cls, sync_id):
         """
         Schedule sync_task to Celery queue.
+
+        :param sync_id: Unique ID for synchronized data object.
         """
-        with transaction.atomic():
-            self.refresh_from_db()
-            self.last_scheduled_dt = timezone.now()
-            self.save()
+        cls.objects.get_or_create(sync_id=sync_id)
+        cls.objects.filter(sync_id=sync_id).update(last_scheduled_dt=timezone.now())
         try:
-            self.sync_task.apply_async(args=[self.sync_id])
+            cls.sync_task.apply_async(args=[sync_id])
         except AttributeError:
             raise NotImplementedError(
                 "Sync task not implemented or not implemented as Celery task."
             )
-        logger.info(f"{self.__class__.__name__} {self.sync_id}: Sync scheduled")
+        logger.info(f"{cls.__name__} {sync_id}: Sync scheduled")
 
-    def started(self, celery_task):
+    @classmethod
+    def started(cls, sync_id, celery_task):
```

Comment:
Observation: This is still theoretically prone to a race condition where multiple workers in parallel pick up the task, from the local perspective of each the task is not started, and both let it start.

Shouldn't be a problem, hopefully:
- I hope that celery is smart enough not to let two workers pick the exact same task at the same time.
- Therefore it could only happen if it was scheduled multiple times. That hopefully doesn't happen normally based on the structure of the callers.